### PR TITLE
fix: Can't make search by email in usersmanagement - EXO-65032 - Meeds-io/meeds#1085

### PR DESCRIPTION
**Pull request by akhanfir**
Prior to this change, when open usersManagement interface and search for users by their mail or login, search shows no results.To fix this problem, added the search by email or login in the search query.After this change, In user management search, should retrieve users by first&last names, emails & usernames.

### CODE REVIEW

**Comment by ahamdi** [comment]
@akhanfir Could you resolve conflicts ?

**Review by rdenarie — CHANGES_REQUESTED** [review]
Can you fix the sonar bug ?
In addition can you try to add a unit test to cover the added lines ?

Thanks

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

File: component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java [hunk 1]
```diff
@@ -338,6 +339,7 @@ public Response getUsers(@Context UriInfo uriInfo,
       Identity target = identityManager.getOrCreateIdentity(OrganizationIdentityProvider.NAME, userId);
       ProfileFilter filter = new ProfileFilter();
       filter.setName(q == null || q.isEmpty() ? "" : q);
+      filter.setSearchEmail(searchEmail ? true : false);
```
Comment (ahamdi) [inline component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java]:
```suggestion
      filter.setSearchEmail(searchEmail);
```

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Review by mkrout — DISMISSED** [review]

**Comment by Jihed525** [comment]
@rdenarie can you have a look ?
FOr test coverage, @akhanfir confirms that it will be handled in the next pr https://github.com/Meeds-io/social/pull/2776

**Review by rdenarie — APPROVED** [review]
ok for me

**Pull request merged**